src/devices/machine/cs8900a.h

```diff
@@ -0,0 +1,114 @@
+// license:GPL-2.0+
+// copyright-holders:Rhett Aultman
+/*************************************************************************
+
+    CS8900A ethernet controller implementation
+
+    by Rhett Aultman <roadriverrail@gmail.com>
+    ported to MAME from VICE Project (https://sourceforge.net/p/vice-emu/)
+    VICE CS8900 code by Spiro Trikaliotis <Spiro.Trikaliotis@gmx.de>
+
+**************************************************************************/
+
+#ifndef MAME_MACHINE_CS8900A_H
+#define MAME_MACHINE_CS8900A_H
+
+#pragma once
+
+#include <queue>
+
+#define CS8900_COUNT_IO_REGISTER 0x10 /* we have 16 I/O register */
+#define MAX_PACKETPAGE_ARRAY 0x1000 /* 4 KB */
```

Comment:
Please don’t leak macros or pollute the global namespace unnecessarily.  You can use `static constexpr` data members for constants.